stable/spark/templates/spark-master-deployment.yaml

```diff
@@ -29,7 +29,7 @@ spec:
       targetPort: {{ .Values.WebUi.ContainerPort }}
   selector:
     component: "{{ .Release.Name }}-{{ .Values.Master.Component }}"
-  type: "LoadBalancer"
+  type: {{ .Values.Master.ServiceType | default "LoadBalancer" }}
```

Comment:
Add the default to `values.yaml` instead of using the `default` function.

Reply:
Done